examples/conftest/README.md

```diff
@@ -0,0 +1,83 @@
+# Conftest Example
+
+This example shows how to set cost policies using [Conftest](https://www.conftest.dev/).  For simplicity, this is based off the terraform-plan-json example, which does not require Terraform to be installed.
```

Comment:
```suggestion
This example shows how to set cost policies using [Conftest](https://www.conftest.dev/).  For simplicity, this is based off the terraform-plan-json example, which does not require Terraform to be installed.

When the policy checks pass, the GitHub Action step called "Check Conftest Policies" passes and outputs `3 tests, 3 passed, 0 warnings, 0 failures, 0 exceptions` in the action logs. When the policy checks fail, that step fails and the action logs show the details of the failing policies.
```

Reply:
Thinking out loud here: it would be great if the Infracost comment can show the cost policy pass/fail line too, maybe this is one of the use-cases for the `get-comment` action, so after conftest runs we can use the `get-comment` action to fetch the last comment, add the conftest results into the markdown file, then use the `comment` action but this time with a markdown file to update the latest comment:

![Screen Shot 2021-12-14 at 4 23 05 PM](https://user-images.githubusercontent.com/638577/146006512-ec2d82bf-c7d4-4ca2-86c1-9be5feac8a9f.png)

Reply:
Yeah that makes a lot of sense because the first thing you're going to do after a policy failure is try and map it back down to the infracost output, and from there map it back to the tf file.  I wonder if we could update the json to include the file/line number of the HCL where the resource was defined.